packages/cli/src/collect/collect.js

```diff
@@ -80,6 +80,12 @@ function buildCommand(yargs) {
       default: 3,
       type: 'number',
     },
+    numberOfFiles: {
+      alias: 'f',
+      description: 'The number of files to test. Disable limit by setting this to 0',
```

Comment:
```suggestion
      description: 'The maximum number of pages to collect when using the staticDistDir option with no specified URLs. Disable this limit by setting to 0.',
```

maybe?

Reply:
Thanks that reads much better, I wasn't sure how long the description may be.